src/forms/types.ts

```diff
@@ -137,3 +140,9 @@ export type RenderType =
   | 'textarea'
   | 'toggle'
   | 'fixed-value';
+
+export interface OHRIFormMarkdown {
```

Comment:
@larslemos you surely don't need this, In my understanding, markdowns are fields or questions, there is an interface for that.

Reply:
If the specification updates, this interface will cover